simulate-event.js

```diff
@@ -1,20 +1,19 @@
-var extend = require('xtend')
 
 /**
  * Set some default options.
  *
  * @type {Object}
  */
 var eventOptions = {
-  UIEvent: function (el) {
+  UIEvent: function () {
     return {
-      view: el.ownerDocument.defaultView
+      view: document.defaultView
```

Comment:
Good point. I wonder if this'll cause any issues. Should we add an option to pass in the related view instead? Just in case. Or, I guess it already can be passed in.